packages/github/package.json

```diff
@@ -38,10 +38,10 @@
     "url": "https://github.com/actions/toolkit/issues"
   },
   "dependencies": {
-    "@actions/http-client": "^1.0.3",
-    "@octokit/core": "^2.5.1",
-    "@octokit/plugin-paginate-rest": "^2.2.0",
-    "@octokit/plugin-rest-endpoint-methods": "^3.10.0"
+    "@actions/http-client": "^1.0.8",
+    "@octokit/core": "^3.0.0",
+    "@octokit/plugin-paginate-rest": "^2.2.3",
+    "@octokit/plugin-rest-endpoint-methods": "^4.0.0"
```

Comment:
Lots of changes to actions api here, going to release a new major version:
https://github.com/octokit/plugin-rest-endpoint-methods.js/releases/tag/v4.0.0